### PR TITLE
add jm comments

### PR DESCRIPTION
**Pull request by jannes-m**
Hey @pat-s ,
I have only slightly updated the text. Hope it helps.
Best,
Jannes

### CODE REVIEW

File: bookdown/07-special-spatiotemp.Rmd [hunk 1]
```diff
@@ -84,6 +86,8 @@ rr_sp = resample(
 rr_sp$aggregate(measures = msr("classif.ce"))
 ```
 
+<!--JM: shouldn't we use `benchmark_grid()` to ensure that the same partitions are used for both resamplings?-->
```
Comment (jannes-m) [inline bookdown/07-special-spatiotemp.Rmd:89]: Shouldn't we use `benchmark_grid()` to ensure that the same partitions are used for both resamplings?

Comment (pat-s) [reply bookdown/07-special-spatiotemp.Rmd:89]: `benchmark_grid()` is just a wrapper around `resample()`. I think calling the same function two times with just a different method is more clear for demonstration purposes. 

Why do you expect the partitions to be equal here? We are using different resampling methods so the partitions will be different.

**Review by pat-s — APPROVED** [review]
Thanks Jannes! Did not expect that :) Seems you are watching closely :)

Reads way better now. My wording was sloppy.

I'll happily merge this and continue with Marcs comments.

File: bookdown/07-special-spatiotemp.Rmd (hunk 1 above)
Comment (jannes-m) [reply bookdown/07-special-spatiotemp.Rmd:89]: You are absolutely right, my bad!

**Pull request merged**